src/DynamoCoreWpf/Views/Search/LibraryView.xaml

```diff
@@ -660,11 +660,38 @@
 
                                         <Border Grid.Row="1"
                                                 Background="#2F2F2F">
-                                            <ItemsPresenter x:Name="ItemsHost"
-                                                            Margin="0,3,0,3">
+                                            <ItemsPresenter x:Name="ItemsHost">
```

Comment:
Can we not use a converter here for the `Margin`? Maybe make a new one that is called `NodeTypeToMarginConverter` that will give the right margin size based on the item type. So this whole thing can be simplified as:

```xml
<ItemsPresenter x:Name="ItemsHost">
    <ItemsPresenter.Margin>
        <Binding Path="..."
                 Converter="{StaticResource NodeTypeToMarginConverter}" />
    </ItemsPresenter.Margin>
</ItemsPresenter>
```

Is this possible?